drivers/hmis/lib/hmis_util/service_types.rb

```diff
@@ -0,0 +1,38 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisUtil
+  class ServiceTypes
```

Comment:
I moved this logic out of the rake task and into a `lib` so that I could run it from the tests and pass an argument (the data source ID). I'm not sure if this is a good place either, but it works.